File: src/models/response/settingsResponse.ts
import {UserResponse} from './userResponse';
export class SettingsResponse {
    Name: string;
    Email: string;
    BirthdayDate: Date;
    IsNotifiable: boolean;
    Blocked: Array<UserResponse>;
    Contacts: Array<any>;
    Genders: Array<any>;
}

File: backend/Vocal.Standard.Model/Response/SettingsResponse.cs
using System;
using System.Collections.Generic;

namespace Vocal.Model.Response
{
    public class SettingsResponse
    {
        public string Name { get; set; }
        public DateTime BirthdayDate { get; set; }
        public string Email { get; set; }
        public List<ChoiceResponse> Genders { get; set; }
        public List<UserResponse> Blocked { get; set; }
        public List<ChoiceResponse> Notifs { get; set; }
        public List<ChoiceResponse> Contacts { get; set; }
        public int TotalDuration { get; set; }
    }
}
